modules/standard/Path.chpl

```diff
@@ -165,14 +154,20 @@ proc absPath(name: string): string throws {
     reliance on :proc:`~FileSystem.locale.cwd()`. Another task on the current
     locale may change the current working directory at any time.
 
-  :return: A normalized, absolutized version of the path for this file.
+  :return: A normalized, absolutized version of the path for the argument file.
```

Comment:
I'd say "file argument" instead of "argument file", personally.  The latter seems to imply that the argument is named file, while the former to me indicates that it is an argument of type file